src/components/FormItem.vue

```diff
@@ -87,21 +97,25 @@ import VALIDATORS from '@/constants/validators';
 
 export default {
     props: {
+        groupName: {
+            type: String,
+            default: '',
+        },
         input: {
             type: Object,
             required: true,
         },
         formErrors: {
             type: [Array, Object],
-            default: () => [],
+            default: () => ([]),
         },
         value: {
             type: String,
             default: '',
         },
         msgRequired: {
```

Comment:
toto by som vyhodil, zbytocny props budeme to definovat vo validators
ak by vadila kompatibilita tak publishneme major verziu

Reply:
Resolved in 41869c9